README.md

```diff
@@ -8,7 +8,7 @@ Rules I have for myself:
 2. Only Atom text editor and Chrome Developer Tools allowed.
 3. SVG use is limited, and all shapes can only use hand-plotted coordinates and bezier curves - without the aid of any graphics editor.
 
-Because of the artistic nature of this project I have not concerned myself with cross-browser-compatibility, so the live preview will most likely look laughable in anything other than chrome.
+Because of the artistic nature of this project I have not concerned myself with cross-browser-compatibility, so the live preview will most likely look laughable in anything other than chrome or Edge.
```

Comment:
Maybe add the words `the latest` to it like `other than the latest Chrome or Edge`?
Latest firefox also seems to work so perhaps it should be added as well?